resnet/README.md

```diff
@@ -22,107 +22,181 @@
-
-# Running a Recipe
-
-Now that we've selected a recipe and determined the `recipe_yaml_path` and `scale_schedule_ratio` to specify, let's kick off a training run.
-
-1. Launch a Docker container using the `mosaicml/pytorch_vision:resnet50_recipes` image on your training system.
-   
-   ```
-   docker pull mosaicml/pytorch_vision:resnet50_recipes
-   docker run -it mosaicml/pytorch_vision:resnet50_recipes
-   ``` 
-   **Note:** The `mosaicml/resnet50_recipes` Docker image can also be used with your container orchestration framework of choice.
-
-1. Download the ImageNet dataset from http://www.image-net.org/.
-
-1. Create the dataset folder and extract training and validation images to the appropriate subfolders.
-   The [following script](https://github.com/pytorch/examples/blob/main/imagenet/extract_ILSVRC.sh) can be used to faciliate this process.
-   Be sure to note the directory path of where you extracted the dataset.
-
-   **Note:** This tutorial assumes that the dataset is installed to the `/tmp/ImageNet` path.
+* Docker image with PyTorch 1.12+, e.g. [MosaicML's PyTorch image](https://hub.docker.com/r/mosaicml/pytorch/tags)
+   * Recommended tag: `mosaicml/pytorch:1.12.1_cu116-python3.9-ubuntu20.04`
+   * The image comes pre-configured with the following dependencies:
+      * PyTorch Version: 1.12.1
+      * CUDA Version: 11.6
+      * Python Version: 3.9
+      * Ubuntu Version: 20.04
+* [Imagenet Dataset](http://www.image-net.org/)
+    * Must be stored either locally (see download instructions [here](https://www.image-net.org/download.php)) or uploaded to an S3 bucket after converting to a [streaming format](https://github.com/mosaicml/streaming) using [this script](https://github.com/mosaicml/streaming/blob/86a9b95189e8b292a8c7880a1c49dc55d1895544/streaming/vision/convert/imagenet.py)
+* System with NVIDIA GPUs
+* Install requirements via `pip install -r requirements.txt` which installs:
+  * [`composer`](https://github.com/mosaicml/composer) - MosaicML's PyTorch training framework
+  * [`streaming`](https://github.com/mosaicml/streaming) - MosaicML's streaming dataset
+  * [`wandb`](https://github.com/wandb/wandb) - Weights and Biases for experiment tracking
+  * [`omegaconf`](https://github.com/omry/omegaconf) - Configuration management
+
+# Dataloader Testing
+
+This benchmark assumes that ImageNet is already stored on your local machine or stored in an S3 bucket after being processed into a streaming dataset. Instructions to download ImageNet can be found [here](https://www.image-net.org/download.php). Alternatively, you can train on other image classification datasets. The local dataset code assumes your data is in the [torchvision ImageFolder](https://pytorch.org/vision/main/generated/torchvision.datasets.ImageFolder.html) format.
```

Comment:
```suggestion
If you want to train on a dataset other than ImageNet, you'll need to either make it a [streaming dataset](https://github.com/mosaicml/streaming) using [this script](https://github.com/mosaicml/streaming/blob/86a9b95189e8b292a8c7880a1c49dc55d1895544/streaming/vision/convert/imagenet.py) or a local [torchvision ImageFolder](https://pytorch.org/vision/main/generated/torchvision.datasets.ImageFolder.html).
```

Reply:
Doesn't this apply to Imagenet as well? So would it be better to say "If you want to train on ImageNet or another dataset..."? I also think it would still be could to include ImageNet links

Reply:
I updated the description with my tweak